api/server/server.go

```diff
@@ -1091,6 +1091,179 @@ func postContainersCopy(eng *engine.Engine, version version.Version, w http.Resp
 	return nil
 }
 
+func handleCopyError(w http.ResponseWriter, statusp *int, errp *error) {
```

Comment:
I think that right now all daemon errors will go through httpError() in server.go - this makes it really easy to ensure we have a consistent (and single place) to deal with all errors.  In particular, I count on this in PR #9509.  By introducing a specialized error method here we'll need to duplicate logic - I'd prefer if this followed the same pattern as all other APIs.